tests/integration/functional/cli/test_hub.py

```diff
@@ -0,0 +1,75 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Test zenml hub CLI commands."""
+
+from click.testing import CliRunner
+
+from zenml.cli.cli import cli
+
+EXAMPLE_PLUGIN_NAME = "langchain_qa_example"
+
+
+def test_hub_list():
+    """Test that zenml hub list does not fail."""
+    runner = CliRunner()
+    list_command = cli.commands["hub"].commands["list"]
+    result = runner.invoke(list_command)
+    assert result.exit_code == 0
```

Comment:
maybe also just check that the `langchain_qa_example` is inside the output string as well? or something else that we know will always be in the hub?